MapboxNavigation/InstructionsCardContainerView.swift

```diff
@@ -66,6 +66,9 @@ public class InstructionsCardContainerView: UIView {
     private var secondaryChildren: [UIView] {
         return [lanesView, nextBannerView]
     }
+
+    @objc dynamic public var customBackgroundColor: UIColor = #colorLiteral(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
```

Comment:
The previous InstructionsCardStyle protocol called this option `backgroundColor`; it controlled the normal background of an unhighlighted card. Did we end up calling it something different to avoid dealing with property overloading in a subclass? I’m not sure this property works; `setGradientLayer(for:)` still calls `backgroundColor` on the style:

https://github.com/mapbox/mapbox-navigation-ios/blob/d0536ead99967cedc97940d367e98f15537d917c/MapboxNavigation/InstructionsCardContainerView.swift#L141

Reply:
Ah, never mind, #2402 takes care of using `customBackgroundColor`. If overloading `backgroundColor` is a problem, then I think `normalBackgroundColor` would be more intuitive in conjunction with the other `highlightedBackgroundColor` property.

Reply:
#2402 has been merged into this PR, so the property is now used. I think it would be great if we could take care of renaming the property to `normalBackgroundColor` before merging.